### PR TITLE
Set CullingController to send on finish cycle event

### LINKED ISSUE #3411: Scene assets are not always loaded when jumping to specific locations
branch for culling controller #3912 

Description:
When loading between locations not all local assets are correctly loaded.

Reproduction steps
- Clear cache
- Switch between different locations and/or jump between local friends
- Notice loading screen can often disappear before all local assets are fully loaded.

Expected results:
Local area assets are loaded before the loading screen is closed.

Notes:
This could again be related to SDK team / message ordering and also potentially be related to #3409 

Platforms
Web Client, Windows Client, MacOS Client.

### PR DESCRIPTION
**Pull request by PalauiDCL**
fixes #3411 

This PR has been reduced to contain only the minimal changes required for calling an end cycle and a restart for the culling controller.

## Use

Get `CullingController `via `ServiceLocator`, ask for a restart of the cycle via `Restart `and subscribe to `CycleFinished`.

### How to test

1. Teleport to any location. The loading screen should fade out as usual
2. Teleport to a World (`/changerealm ile.dcl.eth`). Teleport back to Genesisi City. The loading screen should fade out as usual
3. Do the signup flow. Loading screen should work as usual

### CODE REVIEW

File: unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/LoadingScreen/Scripts/LoadingScreenView.cs [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+        private RenderTexture renderTexture;
+
+        public event Action<ShowHideAnimator> OnFadeInFinish;
+
+        public static LoadingScreenView Create() =>
+            Create<LoadingScreenView>(PATH);
+
+        public override void Start()
+        {
+            base.Start();
```
Comment (davidejensen) [inline unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/LoadingScreen/Scripts/LoadingScreenView.cs:26]: No need of calling the base start method that should be empty in the BaseComponentView

**Review by davidejensen — APPROVED** [review]
LGTM, left just 3 really minor suggestions

File: unity-renderer/Assets/DCLPlugins/ECS7/Systems/ECSSystemsController.cs [hunk 2]
```diff
@@ -48,9 +50,17 @@ public ECSSystemsController(ECS7System componentWriteSystem, SystemsContext cont
         scenesUiDocument.name = "_ECSScenesUI";
         scenesUi = scenesUiDocument.gameObject;
 
+        BaseVariable<bool> loadingScreenVisible;
+
+        if (DataStore.i.featureFlags.flags.Get().IsFeatureEnabled(DataStore.i.featureFlags.DECOUPLED_LOADING_SCREEN_FF))
```
[the remark this answers is not in the document]
Comment (dalkia) [reply unity-renderer/Assets/DCLPlugins/ECS7/Systems/ECSSystemsController.cs:55]: Done!

**Review by kwssbocian — APPROVED** [review]
Issue appears to be fixed
✔️Teleporting to various locations in the normal world as well as teleportation to Worlds via '/changerealm ile.dcl.eth' and back do not produce any issues with the loading screen presentation or behavior.
✔️Signup flow produces no issues with the loading screen as well.
✔️Additionally no issues were observed regarding asset loading on any of the locations.

**Comment by eordano** [comment]
I get  on `/goto -104,-144`

**Comment by eordano** [comment]
Even worse, my mouse becomes unresponsive, as if rendering wouldn't get re-activated

**Comment by dalkia** [comment]
As discussed with @AjimenezDCL and @PalauiDCL, the original request for fixing of the issue is not fixed by using the Restart(). So we are closing this PR with no merge

**Pull request closed without merging**